### PR TITLE
Support for Debian Bullseye / ProxMox 7

### PR DESCRIPTION
**Pull request by imarkov-storpool**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -1,6 +1,11 @@
 - name: Including distribution-specific packages
   include_vars: "{{ ansible_os_family | lower }}.yml"
 
+- name: Including Proxmox-specific packages
+  include_vars: proxmox.yml
+  when:
+    - sp_proxmox_node | bool
```
Comment (netcho) [inline tasks/main.yml:7]: Having a boolean host variable to determine if a host is part of a group of hosts is sub-optimal IMO. I'd suggest checking if a host is part the group `proxmox_hv`, e.g.

File: vars/proxmox.yml
Comment (netcho) [inline vars/proxmox.yml:1]: All packages excluding the `linux-tools` are part of the Debian list of required packages. 
I suppose that replacing `"linux-tools-{{ ansible_kernel }}"` with `"linux-tools-{{ ansible_kernel.split('.')[:2] | join('.') }}"` won't change the behavior of the role? I'd be glad if you can elaborate why Proxmox requires a different Jinja expression. If we can use the Proxmox one to Ubuntu as well, that will simplify the code a lot.

**Review by netcho — CHANGES_REQUESTED** [review]
It looks good to me, it'll be pretty nice to keep it as tidy as possible.

File: tasks/main.yml (hunk 1 above)
Comment (imarkov-storpool) [reply tasks/main.yml:7]: Well, depends on how you define a group... I can change it to look for group membership, all the same :)

File: vars/proxmox.yml
Comment (imarkov-storpool) [reply vars/proxmox.yml:1]: I do need to treat Proxmox nodes a bit differently than regular Debian/Ubuntu. For example the `vlan` package conflicts with `proxmox-ve` and `pve-manager`, and the `linux-tools-common` package does not seem to be present, so they need to be excluded. In fact, I think I had an initial version that did just that. If the `"linux-tools-{{ ansible_kernel.split('.')[:2] | join('.') }}"` expression works on Ubuntu, maybe I can go back to excluding the `vlan` package. But I am not sure that this is really that much simpler than maintaining it as a quasi-separate distribution.

Comment (netcho) [reply vars/proxmox.yml:1]: I understand. It seems reasonable to keep it separate.

File: tasks/main.yml (hunk 1 above)
Comment (netcho) [reply tasks/main.yml:7]: I prefer having a separate group to designate hosts which are Proxmox hypervisors.

**Review by netcho — APPROVED** [review]

**Pull request merged**